### ISSUE TITLE
Show OS distribution stats in the UI

### ISSUE THREAD

**Issue by spderosso** [pypi/warehouse#188]
## Where to show the stats?

Option (a) is in the sidebar after "Downloads Counts", option (b) is to put a link in the sidebar "Advanced Stats" that goes to a separate page where we put all the project stats there.

My only concern with option (a) is that the sidebar might end up being too long in the future, and that if in the future we want to (say) display graphs or stuff like that, then we'll have to do something like option (b). But, maybe we can start with option (a) and move stuff around once we need to? 
## How to show the OS stats?

Option (a) is the raw numbers, option (b) is computing the share. So, option (a) would look something like:

OS Distribution 
400 downloads from Linux
200 downloads from Mac OS
100 downloads from Windows
300 downloads from Other

And option (b) would look something like:

OS Distribution
40% downloads from Linux
20% downloads from Mac
10% downloads from Windows
30% downloads from Other

I don't have any strong opinion here, I think option (a) or (b) would work fine.

**Comment by brainwane** [comment]
@spderosso Thank you for your detailed suggestion, and sorry for the slow response!

For context: The folks working on Warehouse have now gotten [limited funding to concentrate on improving and deploying it](https://pyfound.blogspot.com/2017/11/the-psf-awarded-moss-grant-pypi.html), and have kicked off work towards [our development roadmap](https://wiki.python.org/psf/WarehouseRoadmap) -- the most urgent task is to improve Warehouse to the point where we can redirect pypi.python.org to pypi.org so the site is more sustainable and reliable.

Since this feature, like #699, isn't something that the legacy site has, I've moved it to a future milestone, indicating that the core maintainer team isn't going to work on it till after we've finished shutting down the legacy site. And if I understand correctly, there may be technical or policy difficulties in sniffing out what OS users are using, and in properly indicating that sometimes the sample size is really small or temporally discontinuous in important ways (e.g., if it turns out that most historical data does not have the attributes we'd need to measure). But I'm interested in hearing more about how this could work!

Thanks and sorry again for the wait.
